library/src/com/google/maps/android/geojson/GeoJsonGeometryCollection.java

```diff
@@ -0,0 +1,53 @@
+package com.google.maps.android.geojson;
+
+import java.util.ArrayList;
+
+/**
+ * A GeoJsonGeometryCollection geometry contains a number of GeoJsonGeometry objects.
+ */
+public class GeoJsonGeometryCollection implements GeoJsonGeometry {
+
+    private final static String GEOMETRY_TYPE = "GeometryCollection";
+
+    private final ArrayList<GeoJsonGeometry> mGeometries;
+
+    /**
+     * Creates a new GeoJsonGeometryCollection object
+     *
+     * @param geometries array of GeoJsonGeometry objects to add to the GeoJsonGeometryCollection
+     */
+    public GeoJsonGeometryCollection(
+            ArrayList<GeoJsonGeometry> geometries) {
+        if (geometries == null) {
+            throw new IllegalArgumentException("Geometries cannot be null");
+        }
+        mGeometries = geometries;
+    }
+
+    /**
+     * Gets the type of geometry
```

Comment:
Since this is the same as the parent, I think you can omit the javadoc completely. Or use @inheritdoc or something like that.